Stratis.Bitcoin/Features/MemoryPool/MempoolFeature.cs

```diff
@@ -8,15 +8,57 @@
 
 namespace Stratis.Bitcoin.Features.MemoryPool
 {
+    /// <summary>
+    /// Transaction memory pool feature for the Full Node.
+    /// <see cref="https://github.com/bitcoin/bitcoin/blob/6dbcc74a0e0a7d45d20b03bb4eb41a027397a21d/src/txmempool.cpp"/>
```

Comment:
This should be `<seealso>` outside summary. `<see>` is inline construct and is used slightly differently.